### PR TITLE
UIU-1352: Update expand/collapse filter pane button implementation

### PR DESCRIPTION
**Pull request by rasmuswoelk**
Updated expand/collapse search pane functionality to be the same as in `<SearchAndSort>`.

### CODE REVIEW

**Comment by rasmuswoelk** [comment]
Note: Tests are working fine in an independent clone but _not_ in my workspace locally.

And it looks like they are failing in the CI as well.

File: src/views/UserSearch/UserSearch.js [hunk 1]
```diff
@@ -330,7 +318,15 @@ class UserSearch extends React.Component {
                     {intl => (
                       <Paneset id={`${idPrefix}-paneset`}>
                         {this.state.filterPaneIsVisible &&
-                          <Pane defaultWidth="22%" paneTitle="User search">
+                          <Pane
+                            defaultWidth="22%"
+                            paneTitle="User search"
```
Comment (VictorSoroka1) [inline src/views/UserSearch/UserSearch.js]: Please translate this.

File: src/views/UserSearch/UserSearch.js [hunk 2]
```diff
@@ -144,31 +145,18 @@ class UserSearch extends React.Component {
...
-                  aria-label={`${hideOrShowMessage} \n\n${appliedFiltersMessage}`}
-                  onClick={this.toggleFilterPane}
-                  badge={!filterPaneIsVisible && filterCount ? filterCount : undefined}
-                />
-              )}
-            </FormattedMessage>
-          )}
-        </FormattedMessage>
+        <ExpandFilterPaneButton
+          onClick={this.toggleFilterPane}
```
Comment (VictorSoroka1) [inline src/views/UserSearch/UserSearch.js]: Please move callbacks to the bottom for a better readability and scanability.

Comment (rasmuswoelk) [reply src/views/UserSearch/UserSearch.js]: Done :)

File: src/views/UserSearch/UserSearch.js (hunk 1 above)
Comment (rasmuswoelk) [reply src/views/UserSearch/UserSearch.js]: Done :)

**Pull request merged**